extensions/jupiter-aggregator/src/compare-token-prices.tsx

```diff
@@ -0,0 +1,80 @@
+import { Detail, useNavigation, Clipboard, Action, ActionPanel, Form, showToast, Toast } from "@raycast/api";
+import { fetchTokenList, compareTokens } from "./api";
+import { useState } from "react";
+
+function Result({ price }) {
+  const markdown = `
+# Result:
+# ${price}
```

Comment:
For extra context, can we add the token name in the result? Eg: `25.5 USDC`

Reply:
Excellent idea!! Done!